dataland-frontend/src/components/generics/ViewMultipleDatasetsDisplayBase.vue

```diff
@@ -41,7 +41,7 @@
               v-if="frameworkViewConfiguration?.type == 'MultiLayerDataTable'"
               :frameworkIdentifier="dataType"
               :companyId="companyId"
-              :displayConfiguration="frameworkViewConfiguration!!.configuration"
+              :display-configuration="frameworkViewConfiguration!!.configuration"
```

Comment:
warum hat sich die schreibweise verändert?

Reply:
Es funktioniert mit beiden, so wie das Framework-Toolbox konzipiert wurde. Aber display-configuration wird in den anderen nicht automatiserte Frameworks (financials, p2p) verwendet, da das vue standard notation ist. Deshalb die Änderung.